src/app/app.module.ts

```diff
@@ -16,6 +19,14 @@ import { InventoryData } from '../providers/inventory-data';
 import { StockpileData } from '../providers/stockpile-data';
 import { UserData } from '../providers/user-data';
 
+let storage = new Storage();
+
+export function getAuthHttp(http) {
+  return new AuthHttp(new AuthConfig({
+    tokenGetter: (() => storage.get('id_token')),
```

Comment:
Why use snake case here for `id_token`? Shouldn't it be `idToken`?

(Used in a few places in the user data provider as well.)